mentoring/questionnaire.py

```diff
@@ -126,3 +130,14 @@ def get_submission_display(self, submission):
             if choice.value == submission:
                 return choice.content
         return submission
+
+    def uuid(self):
```

Comment:
@mtyaka looks pretty much like [lazy](https://pypi.python.org/pypi/lazy) property:

    @lazy
    def uuid(self):
        return uuid.uuid().hex

Accessing later would look like `block.uuid` (not no braces)